### DISCUSSION TITLE
where is the application available after successful publishing?

### DISCUSSION THREAD

**Discussion by MatuszykEryk** [Triple-T/gradle-play-publisher#1019] [Q&A] [answered]
Hi.
I was able to fix some configuration bugs.
currently using the following commands:
app:uploadReleasePrivateBundle or app:uploadReleasePrivateBundle gets a positive result:
```
> Task: app: uploadReleasePrivateBundle
Starting App Bundle upload: C: \ TeamCity \ buildAgent \ work \ 886f965d5e1e01a4 \ android \ app \ build \ outputs \ bundle \ release \ app-release.aab
Uploading App Bundle: 40% complete
Uploading App Bundle: 80% complete
App Bundle upload complete
Upload successful: https://play.google.com/apps/test/xxxx/xxxxxxxxxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxxxx
```
I assume that the positive result of uploading the files suggests that the version was correctly uploaded.
my build.gradle configuration
```
play {
  serviceAccountCredentials.set (file ('C: \\ key.json'))  
 track.set ("internal")
}
```
seeing the upload progress, I assume that the application is correctly published on internal tests.
suggests that the upload should take place on the inner track. after uploading the package, the new version is not visible in play console.
I will add that after a successful build result, the version is not visible in the play store library. why?

**Comment by SUPERCILEX** [comment, marked as answer]
See https://github.com/Triple-T/gradle-play-publisher/issues/1002#issuecomment-915794828

**Reply by MatuszykEryk** [reply]
it was enough to use the gradlew app command: publishReleaseBundle
Thank you very much for help.
